dotcom-rendering/src/lib/articleCount.ts

```diff
@@ -37,15 +38,22 @@ export const getArticleCounts = async (
 	// hasOptedOut needs to be done before we check if articleCount is set in the window
 	// This is because a potential race condition where one invocation of getArticleCounts
 	// is waiting for hasOptedOut another invocation might receive it and increment the article count.
+
+	const keywordAndToneTagIds: string[] = tags.flatMap((tag) =>
```

Comment:
interesting use of `flatMap`! Is this for efficiency?

Reply:
This was something that I tried to do (more like a way to solve it by myself  and learn ) , I understand it is different from  the code change suggested  earlier, is this  wrong?Just wanted to check .I am not sure about the efficiency.I will put the suggested code.